server/routes/bookAVisit.ts

```diff
@@ -207,6 +207,16 @@ export default function routes(
       if (!Object.keys(formValues).length && visitSessionData.visit?.id) {
         formValues['visit-date-and-time'] = visitSessionData.visit?.id
       }
+      let slotsPresent = false
+
+      Object.entries(slotsList).find(([, value]) => {
+        if (value.length !== 0) {
+          slotsPresent = true
+          return false
+        }
+        return true
+      })
+      slotsPresent = Object.values(slotsList).some(value => value.length)
```

Comment:
do you need the section above now you've added this line?

Reply:
well spotted I had it folded in my view...

Reply:
👍 can probably just do `const slotsPresent = Object....` rather than `let` as you don't re-assign it.

Reply:
got this implemented! much cleaner data